### ISSUE TITLE
Init project failed

### ISSUE THREAD

**Issue by Trojan0523** [sanyuan0704/island.js#57]
### Describe the bug

  according to getting started,  using pnpm to init the island-app project, then start dev server using "pnpm run dev", terminal output  `Error [ERR_MODULE_NOT_FOUND]` 
it seems like it is a small bug that needs me to install unocss as devDependencies additionally

### Reproduction

- Step1: mkdir island-app && cd island-app
- Step2: npm init -y
- Step3: pnpm add islandjs
- Setp4: mkdir docs && echo '# Hello World' > docs/index.md
- Step5: add `islan dev docs` as dev-script  in `package.json`
- Step6: pnpm run dev
- reproduce 

### Expected behavior

start the dev server according to the [Getting Started Page](https://island.sanyuan0704.top/en/guide/getting-started.html) without any additional behavior

### System Info

```Shell
System:
    OS: macOS 12.4
    CPU: (8) x64 Apple M1
    Memory: 18.11 MB / 16.00 GB
    Shell: 5.8.1 - /bin/zsh
  Binaries:
    Node: 16.4.0 - ~/.nvm/versions/node/v16.4.0/bin/node
    Yarn: 1.22.10 - ~/.nvm/versions/node/v14.6.0/bin/yarn
    npm: 7.18.1 - ~/.nvm/versions/node/v16.4.0/bin/npm
  Browsers:
    Chrome: 105.0.5195.125
    Firefox: 104.0.2
    Safari: 15.5
  npmPackages:
    islandjs: ^0.2.6 => 0.2.6
```

### Validations

- [X] Read the [docs](https://island.sanyuan0704.top)
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.

**Comment by Trojan0523** [comment]
add on: Error Message
```
(node:8879) UnhandledPromiseRejectionWarning: Error [ERR_MODULE_NOT_FOUND]: Cannot find package 'unocss' imported from /Users/buxiongyu/Desktop/demo/island-app/node_modules/.pnpm/islandjs@0.2.6/node_modules/islandjs/dist/node/chunk-ILP4AUY6.js
    at packageResolve (internal/modules/esm/resolve.js:674:9)
    at moduleResolve (internal/modules/esm/resolve.js:715:14)
    at Loader.defaultResolve [as _resolve] (internal/modules/esm/resolve.js:808:11)
    at Loader.resolve (internal/modules/esm/loader.js:82:40)
    at Loader.getModuleJob (internal/modules/esm/loader.js:226:28)
    at ModuleWrap.<anonymous> (internal/modules/esm/module_job.js:51:40)
    at link (internal/modules/esm/module_job.js:50:36)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:8879) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 20)
```

**Comment by sanyuan0704** [comment]
ok, i will fix it right away.

**Comment by sanyuan0704** [comment]
I have solved it in v0.2.7

**Comment by sanyuan0704** [comment]
The reason is that `unocss` added in `devDependenceis` so i take it in `dependencies` instead.

**Comment by Trojan0523** [comment]
> I have solved it in v0.2.7

thank you so much
